tracer/test/test-applications/integrations/Samples.Wcf/App.config

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<configuration>
+  <startup>
+        <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.6.2" />
+    </startup>
+  <appSettings>
+  </appSettings>
+  <system.diagnostics>
+    <sources>
+      <source name="System.ServiceModel"
```

Comment:
These aren't required right, just helpful for testing?

Reply:
oops. Relicate of a previous test. 
Also, it's possible to do the injection differently, using an extension that you would define in the app.config. So wanted to test it later on. As it would be simpler for customers than creating spans in many different places potentially